fast/stages/1-resman/variables.tf

```diff
@@ -227,16 +225,14 @@ variable "prefix" {
 variable "tag_names" {
   description = "Customized names for resource management tags."
   type = object({
-    context      = string
-    environment  = string
-    org-policies = string
-    tenant       = string
+    context     = string
+    environment = string
+    tenant      = string
   })
   default = {
-    context      = "context"
-    environment  = "environment"
-    org-policies = "org-policies"
-    tenant       = "tenant"
+    context     = "context"
+    environment = "environment"
+    tenant      = "tenant"
```

Comment:
I'd move the default values to the fields
```hcl
type = object({
    context     = optional(string, "context")
    environment = optional(string, "environment")
    tenant      = optional(string, "tenant")
  })
  default = {}
  nullable = false
```

Reply:
actually as it is right now you have to provide all the values if you want to change a single one.

Reply:
gosh, swapped your code in the comment with my code, let me fix great suggestion :)